### ISSUE TITLE
can only assign merge request to the owner of the merge request

### ISSUE THREAD

**Issue by tonicbupt** [gitlabhq/gitlabhq#9247]
version 7.10, after upgrading from 7.9, I found that I can't choose anyone else but myself.

the query seems to be empty.

```
GET /autocomplete/users.json?search=w&per_page=20&active=true&project_id=865
[]
```

if I remove project_id from the query the result is not empty, so where did I do wrong?

![image](https://cloud.githubusercontent.com/assets/1748597/7489896/1bf8bf98-f40e-11e4-9319-2648da596e63.png)

**Comment by gaowhen** [comment]
It's not your fault, we too confront this problem after upgrading to version 7.10.

**Comment by tonicbupt** [comment]
@gaowhen how did you find this!!!

**Comment by remcoros** [comment]
Something simular happens with adding new users to groups (which isn't possible at the moment) see #9261

**Comment by stanhu** [comment]
I believe this is fixed by c4e48d27a7922490.

/cc: @Razer6

**Comment by Razer6** [comment]
Then let's close it. Thanks!
